docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -1,8 +1,8 @@
 # Get invited and onboard to TechPass
 
-This article guides you how to request for a TechPass account and onboard to it. Refer to the flow chart on [Onboard to TechPass](onboard-to-techpass) page to know if you can sign up for TechPass account via TechPass portal or not. If you can't sign up for TechPass account via TechPass portal, you need to request to get invited.
+This article walks you through the TechPass portal sign-up and onboarding process. Check the flow chart on the [Onboard to TechPass](onboard-to-techpass) page to see if you're eligible to sign up via the portal. If you can't sign up for TechPass account via TechPass portal, you need to request to get invited.
```

Comment:
flowchart is 1 word.. 
can i suggest.. "You can use the flowchart, on the ... page to check if you are eligible to sign up via the TechPass Portal."?

when do you use ' and when do you spell in full?

request from who? or how to request?

Reply:
fixed